src/librustc/infer/outlives/verify.rs

```diff
@@ -141,9 +141,9 @@ impl<'cx, 'gcx, 'tcx> VerifyBoundCx<'cx, 'gcx, 'tcx> {
     }
 
     fn recursive_type_bound(&self, ty: Ty<'tcx>) -> VerifyBound<'tcx> {
-        let mut bounds = ty.walk_shallow()
+        let mut bounds: Vec<_> = ty.walk_shallow()
```

Comment:
we don't have a style guide for this and preferences will go either way. Thus, please don't change existing code and change this back.

Reply:
Okay. I just went with the apparent (overwhelming) convention, but fine...